src/pages/EditRequestAmountPage.js

```diff
@@ -17,9 +18,12 @@ const propTypes = {
 
     /** Callback to fire when the Save button is pressed  */
     onSubmit: PropTypes.func.isRequired,
+
+    /** reportID for the "transaction thread" */
```

Comment:
I think this is commonly used term now
```suggestion
    /** reportID for the transaction thread */
```